src/System.Runtime.WindowsRuntime/src/System/Resources/WindowsRuntimeResourceManager.cs

```diff
@@ -231,7 +231,20 @@ private static void GlobalResourceContextChanged(object sender, IMapChangedEvent
             Debug.Assert(s_globalResourceContextFallBackList != null);
             Debug.Assert(s_globalResourceContext != null);
 
-            List<String> languages = new List<string>(s_globalResourceContext.Languages);
+            IReadOnlyList<string> langs;
+
+            try
+            {
+                langs = s_globalResourceContext.Languages;
+            }
+            catch (ArgumentException)
+            {
+                // Sometimes Windows Runtime fails and we get Argument Exception which can fail fast the whole app
+                // to avoid that we ignore the exception.
```

Comment:
@stephentoub are you ok to merge this one or you still want to have todo tracking issue? I am doing more investigation on Windows side anyway.